app/assets/javascripts/manage_submissions.js

```diff
@@ -1,4 +1,171 @@
-$(document).ready(function() {
+const manage_submissions_endpoints = {
+  'score_details': 'submissions/score_details',
+}
+
+function get_score_details(course_user_datum_id) {
+  return new Promise((resolve, reject) => {
+    $.ajax({
+      url: manage_submissions_endpoints['score_details'],
+      type: 'GET',
+      data: { cuid: course_user_datum_id },
+      success: function (data) {
+        resolve(data);
+      },
+      error: function (err) {
+        console.log(err);
+        reject(err);
+      }
+    })
+  });
+}
+
+$(document).ready(function () {
+
+  $('.modal').modal();
+
+  $('.score-details').on('click', function () {
+    // Get the email 
+    const course_user_datum_id = $(this).data('cuid');
+    const email = $(this).data('email');
+
+    // Set the email
+    $('#score-details-email').html(email);
+
+    // Clear the modal content
+    $('#score-details-content').html('');
+
+    // Add a loading bar
+    $('#score-details-content').html(`
+        <div class="progress">
+            <div class="indeterminate"></div>
+        </div>`);
+
+    // Open the modal
+    $('#score-details-modal').modal('open');
+
+    // Fetch data and render it in the modal 
+    get_score_details(course_user_datum_id).then((data) => {
+      console.log(data);
+
+      const problem_headers = data.submissions[0].problems.map((problem) => {
+        const max_score = problem.max_score;
+        const autograded = problem.grader_id < 0 ? " (Autograded)" : "";
+        return `<th class="submission-th">
+                  ${problem.name}
+                  <br>
+                  ${max_score} ${autograded}
+                </th>`;
+      }).join('');
+
+      const submissions_body = data.submissions.map((submission) => {
+        
+        let tweak_value = data?.tweaks[submission.id]?.value ?? "None";
+        if (tweak_value != "None" && tweak_value > 0) {
+          tweak_value = `+${tweak_value}`;
+        }
+
+        // Convert to human readable date with timezone 
+        const human_readable_created_at = moment(submission.created_at).format('MMM Do YYYY, h:mm:ss a z Z');
```

Comment:
Is this the finalized date format? I remember in the team meeting we discussed a specific format but don't remember what the final format we decided on was.

Reply:
I updated it to match